Source/WebCore/workers/WorkerGlobalScope.cpp

```diff
@@ -158,7 +159,7 @@ void WorkerGlobalScope::prepareForDestruction()
 {
     WorkerOrWorkletGlobalScope::prepareForDestruction();
 
-    removeSupplement("WorkerGlobalScopeTrustedTypes"_s);
+    removeSupplement(WindowOrWorkerGlobalScopeTrustedTypes::workerGlobalSupplementName());
```

Comment:
Ryosuke will perf A/B test a patch to make this less error prone. Hopefully, perf is OK and we can get rid of this class of bugs.